### PR TITLE
Change coordinates from (x, y) format to (row, col) format

### PR DESCRIPTION
**Pull request by Meng87**
- changes format
- fixes initial bug where grid rotates

### CODE REVIEW

File: src/components/GridBoard/GridBoard.js [hunk 1]
```diff
@@ -136,12 +136,7 @@ export default function GridBoard(props) {
         console.log("Terraform value greater than 10")
```
Comment (ppalwai2001) [inline src/components/GridBoard/GridBoard.js:136]: Shouldn't this be ~10+1, since ~10+1 = -10

File: src/components/GridBoard/GridBoard.js [hunk 2]
```diff
@@ -359,10 +344,10 @@ export default function GridBoard(props) {
             console.log("Terraform value greater than 10")
```
Comment (ppalwai2001) [inline src/components/GridBoard/GridBoard.js:344]: Like above, this should ~10+1 instead of ~10

**Review by ppalwai2001 — CHANGES_REQUESTED** [review]
The files look great in terms of using (row,col) format instead of (x,y). Just minor comments on checking using ~10 +1 instead of ~10, since ~10 = -11

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Pull request merged**